gossip/convergence_test.go

```diff
@@ -45,6 +46,5 @@ func verifyConvergence(numNodes, maxCycles int, t *testing.T) {
 // actual production gossip code than seems worthwhile for a unittest.
 func TestConvergence(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	t.Skipf("TODO(pmattis): #4500")
 	verifyConvergence(10, 100, t)
```

Comment:
How many iterations are now required? Can we lower the "100" here? (it was 20 until a few weeks ago)